client/state/analytics/actions.js

```diff
@@ -81,3 +84,8 @@ export const recordPageView = ( url, title, service ) => ( {
 
 export const recordGooglePageView = ( url, title ) =>
 	recordPageView( url, title, 'ga' );
+
+export const startContinuousTracking = ( trackingTool ) => ( {
+	type: ANALYTICS_CONTINOUS_MONITOR,
+	payload: loadTrackingTool( trackingTool ),
```

Comment:
`loadTrackingTool` shouldn't live in payload. Instead, you should listen for `ANALYTICS_CONTINUOUS_MONITOR` in [this `switch` statement](https://github.com/Automattic/wp-calypso/blob/b1479afaef85acc62ecba8197316f620510ebf96/client/state/analytics/middleware.js#L27-L36) and run it there.